libs/ui/label/helm/src/lib/hlm-label.directive.ts

```diff
@@ -46,27 +46,20 @@ export class HlmLabelDirective {
 	private readonly _brn = inject(BrnLabelDirective, { host: true });
 
 	public readonly userClass = input<ClassValue>('', { alias: 'class' });
+
+	public readonly variant = input<LabelVariants['variant']>('default');
+
+	public readonly error = model<LabelVariants['error']>('auto');
```

Comment:
What is a better approach to set `error` programmatically? I changed it from `input` to `model` to be able to use `.set`.
`input-error.directive.ts` is setting the error state programmatically not through template bindings.

Reply:
In future we will be able to use a `linkedSignal` for something like this, however for the time being we can use a computed:

```ts
public readonly error = input<LabelVariants['error']>('auto');

protected readonly state = computed(() => ({
    error: signal(this.error())
}));

protected readonly _computedClass = computed(() =>
		hlm(
			labelVariants({
				variant: this.variant(),
				error: this.state().error(),
				disabled: this._brn?.dataDisabled() ?? 'auto',
			}),
			'[&.ng-invalid.ng-touched]:text-destructive',
			this.userClass(),
		),
	);

setError(error: LabelVariants['error']): void {
   this.state().error.set(error);
}
```

This way we can keep the `error` as just an `input` which is more appropriate, but also update the error internally. This allows the input to take priority as any time it changes it will generate a new error signal in the `state` computed signal, but still allows us to modify it without needing to use the input when we need.